### DISCUSSION TITLE
6.7 released, question about docker images

### DISCUSSION THREAD

**Discussion by eddi-exp** [docker-library/wordpress#929] [General]
Hello,

I am fairly new to the docker hub images. The other day I noticed my build with wordpress:latest brought WordPress 6.7 online. It did upset my containers, as one was running 6.6.2 and one was on 6.7. I noticed a few older plugins I was testing with throw errors with 6.7

Now to the question: When I'm reviewing Docker Hub I don't see a stable release of 6.7 yet, how far behind is the docker hub images from stable release? I do see beta-6.7 listed towards the bottom. I would want to be on an image like beta-6.7-php8.2-apache. I don't want to set the containers to wordpress:latest again in fear it would deploy a new patch automatically during the night. Should I just stick with the beta images for now? How soon would there be a stable release of the other docker image flavors?

https://hub.docker.com/_/wordpress

Edit: I'm still not sure how wordpress 6.7 got onto one of two containers, perhaps the older container did an automatic core update while the newer container over night rebuilt with 6.6.2. I've since disabled core updates with:

`define( 'WP_AUTO_UPDATE_CORE', false );`

**Comment by tianon** [comment]
As you've noted, WordPress itself does automatic updates, and I'd honestly recommend using them, but it sounds like you've got a bit of a unique setup with multiple WordPress instances collaborating somehow?

Either way, 6.7 GA was released, as seen in https://github.com/docker-library/wordpress/commit/805a50001c4457b69c949cbc8b2ff1f7e563e2c6 :+1:
